### PR TITLE
findbugs plugin has been updated. 2 errors were solved

### PR DESCRIPTION
**Pull request by recena**
@reviewbybees

### CODE REVIEW

**Comment by andresrc** [comment]
:bee:

File: src/test/java/hudson/scm/SubversionSCMTest.java [hunk 1]
```diff
@@ -97,8 +97,8 @@ protected void setUp() throws Exception {
         super.setUp();
         repo = SVNURL.parseURIDecoded("http://subversion.tigris.org/svn/subclipse");
 
-        // during the test, don't pollute the global authentication cache.
-        SubversionSCM.CONFIG_DIR = createTmpDir().getAbsolutePath();
+        // during the test, don't pollute the user's configuration (esp. authentication cache).
+        System.setProperty(SubversionSCM.class.getName() + ".configDir", createTmpDir().getAbsolutePath());
```
Comment (amuniz) [inline src/test/java/hudson/scm/SubversionSCMTest.java]: Is this already visible on `SubversionSCM ` class loading?

**Comment by amuniz** [comment]
:bee:

File: src/test/java/hudson/scm/SubversionSCMTest.java (hunk 1 above)
Comment (recena) [inline src/test/java/hudson/scm/SubversionSCMTest.java]: Really, in this point you define the environment variable, only that. When `SubversionSCM` is instantiated, it hopes that `SubversionSCM.CONFIG_DIR` have this value for the tests. [Reference](https://github.com/jenkinsci/subversion-plugin/blob/master/src/main/java/hudson/scm/SubversionSCM.java#L3149) happens.

**Comment by recena** [comment]
@reviewbybees done

File: src/test/java/hudson/scm/SubversionSCMTest.java (hunk 1 above)
Comment (recena) [inline src/test/java/hudson/scm/SubversionSCMTest.java]: And yes, `SubversionSCM` class is loaded in this poing.

**Pull request merged**